modules/puzzles/module.py

```diff
@@ -60,14 +59,44 @@ def coordinates(self, numbers):
         """
         Geeft alle mogelijke locaties op de campus met coordinaten bestaande uit de gegeven getallen. Getallen dienen te worden gescheiden met komma's
         """
-        results = util.brute_force_coordinates(numbers.split(','))
-        self.respond('Er zijn %i locaties op de campus met deze getallen.' % (len(results)))
-        if(results):
-            if len(results) <= 5:
-                for result in results:
-                    self.send_location(result[0], result[1])
-            elif len(results) <= 15:
+        number_list = numbers.split(',')
+        if all([x.isdigit() for x in number_list]):
+            results = util.brute_force_coordinates(number_list)
+            self.respond('Er zijn %i locaties op de campus met deze getallen.' % (len(results)))
+            if(results):
                 self.respond('\n'.join(['%s, %s' % x for x in results]))
```

Comment:
Stuur je nou geen locations meer terug?

Reply:
Dat is niet de bedoeling. Ik kan niet vinden wat er fout is gegaan.